private/buf/cmd/buf/buf.go

```diff
@@ -287,7 +289,8 @@ func wrapError(err error) error {
 				"visit https://docs.buf.build/bsr/authentication`, authErr.TokenEnvKey())
 			}
 			return errors.New(`Failure: you are not authenticated. Create a new entry in your netrc, " +
-			"using a Buf API Key as the password. For details, visit https://docs.buf.build/bsr/authentication`)
+				"using a Buf API Key as the password. If you already have an entry in your netrc, check " +
+				"to see that your token is not expired. For details, visit https://docs.buf.build/bsr/authentication`)
```

Comment:
maybe dumb question, but given that we're using raw strings here, wouldn't they include the raw `"` and `+` characters here? (Going to check locally.)

Reply:
ya, let me fix that, heh:

```console
$ buf beta registry repository get buf.build/svanburen/test-private
Failure: you are not authenticated. Create a new entry in your netrc, " +
				"using a Buf API Key as the password. If you already have an entry in your netrc, check " +
				"to see that your token is not expired. For details, visit https://docs.buf.build/bsr/authentication
```

Reply:
Fixed in https://github.com/bufbuild/buf/pull/2636/commits/cdad00cc70d50f750d61881213d174a8ad2c80a2.